client/src/components/Languages.jsx

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { TableHeader, LangContainer, TableData, TableTitle, TableWrapper, TableRow } from './styles/styles';
+
+const Languages = ({ languages }) => (
+  <div>
+    <LangContainer>
+      <TableTitle>Languages:</TableTitle>
+      <TableWrapper>
+        <tbody>
+          <TableRow>
+            <TableHeader> </TableHeader>
+            <TableHeader>Interface</TableHeader>
+            <TableHeader>Full Audio</TableHeader>
+            <TableHeader>Subtitles</TableHeader>
+          </TableRow>
+          <TableRow>
+            <td>{languages[0]}</td>
+            {languages[1] ? <TableData>&#10004;</TableData> : <TableData />}
```

Comment:
Good job checking for data before you render! Similar to what Leslie mentioned tonight - may want to have default data displayed if there is none? I need to do that as well if I'm being honest but just a suggestion

Reply:
Just fyi, this is not meant to render anything if it is false. It is an item that is only there sometimes, so this was fully intentional.